lib/jarbs/project_generator.rb

```diff
@@ -33,6 +33,11 @@ def generate
 
       write_package(manifest, @name)
       NodeBuild.new(nil).npm_install(@name)
+      install_handler
+    end
+
+    def install_handler
```

Comment:
Ah, I thought install_handler meant 'the handler for the install [of fixtures]'. Derp. >.<